src/shared/components/flexbox/FlexBox.tsx

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
Lets look to see if this can be accomplished with Bootstrap grid.  we want to maintain a bias toward bootstrap for layout.